pkg/config/autodiscover/autodiscover_partner.go

```diff
@@ -30,7 +30,7 @@ const (
 )
 
 func IsMinikube() bool {
```

Comment:
Why not renaming this function to IsNonOcpCluster()?

Reply:
done